reconnecting-eventsource.d.ts

```diff
@@ -0,0 +1,21 @@
+export interface Configuration extends EventSourceInit {
+    // the maximum time to wait before attempting to reconnect in ms, default `3000`
+    // note: wait time is randomised to prevent all clients from attempting to reconnect simultaneously
+    max_retry_time: number;
```

Comment:
Isn't this optional?

```suggestion
    max_retry_time?: number;
```

Reply:
updated